gnuradio-runtime/lib/basic_block.cc

```diff
@@ -201,7 +202,8 @@ namespace gr {
     gr::thread::scoped_lock guard(mutex);
 
     if((msg_queue.find(which_port) == msg_queue.end()) || (msg_queue_ready.find(which_port) == msg_queue_ready.end())) {
-      std::cout << "target port = " << pmt::symbol_to_string(which_port) << std::endl;
+      GR_LOG_DEBUG (d_logger, "target port = " << pmt::symbol_to_string(which_port));
```

Comment:
So, I will also add a GR_LOG_ERROR wherever `throw` is used, But I think this debugging message is correct.

Also, I tried to duplicate both because we can configure both `d_logger` and `d_debug_logger` independently. For an example: If I like to view warning level messages on console and only "info" level messages in the file, I can accordingly configure both independently. Hence, giving all messages to both of them will ensure independent configurations of both. Is there something I am missing?